GEON-PPANG-iOS/Presentation/Common/StackView/BreadTypeStackView.swift

```diff
@@ -87,10 +87,10 @@ final class BreadTypeStackView: UIStackView {
         } else {
             nutFreeChip.isHidden = false
         }
-        if !isnoSugarChip {
-            noSugarChip.isHidden = true
+        if !issubSugarChip {
+            subSugarChip.isHidden = true
```

Comment:
p4:
```suggestion
            subSugarChip.isHidden = !issubSugarChip
```
처럼 나타낼 수 있습니당

Reply:
[ba0c8b2](https://github.com/GEON-PPANG/GEON-PPANG-iOS/pull/180/commits/ba0c8b213eb9b576b99ccf5b82a3c06b3d4f3f12)